actions/websocket_actions.jsx

```diff
@@ -437,11 +439,11 @@ export function handleEvent(msg) {
         break;
 
     case SocketEvents.HELLO:
-        handleHelloEvent(msg);
+        wrapEventWithJitter(handleHelloEvent, msg);
```

Comment:
I think this one is only sent to one user at a time on their first connection or on reconnect if there were missed messages. I don't know why it involves fetching roles, but that probably shouldn't be part of `setServerVersion`

Reply:
removed